shell/platform/darwin/common/framework/Source/FlutterNSBundleUtils.h

```diff
@@ -30,6 +30,10 @@ NSBundle* FLTFrameworkBundleInternal(NSString* bundleID, NSURL* searchURL);
 // `+[NSBundle bundleWithIdentifier:]`.
 NSBundle* FLTFrameworkBundleWithIdentifier(NSString* bundleID);
 
+NSBundle* FLTGetApplicationBundle();
+
+NSURL* FLTAssetsFromBundle(NSBundle* bundle);
```

Comment:
Can this be given a more precise name? I'm assuming this is an assets directory, rather than actual assets?